mdp_playground/scripts/run_experiments.py

```diff
@@ -0,0 +1,221 @@
+'''Script to run experiments on MDP Playground.
```

Comment:
I don't think I want these scripts in a separate directory. For one thing, scripts is a vague name imo.

Reply:
Could you please undo the scripts part?